include/z64pause.h

```diff
@@ -63,22 +63,22 @@ typedef enum PauseState {
     /*  5 */ PAUSE_STATE_OPENING_2, // Finish some animations for opening the menu.
     /*  6 */ PAUSE_STATE_MAIN, // Pause menu ready for player inputs.
     /*  7 */ PAUSE_STATE_SAVE_PROMPT,  // Save prompt in the pause menu
-    /*  8 */ PAUSE_STATE_8,
-    /*  9 */ PAUSE_STATE_9,
-    /* 10 */ PAUSE_STATE_10,
-    /* 11 */ PAUSE_STATE_11,
-    /* 12 */ PAUSE_STATE_12,
-    /* 13 */ PAUSE_STATE_13,
-    /* 14 */ PAUSE_STATE_14,
-    /* 15 */ PAUSE_STATE_15,
-    /* 16 */ PAUSE_STATE_16,
-    /* 17 */ PAUSE_STATE_17,
+    /*  8 */ PAUSE_STATE_GAME_OVER_REQUEST,
+    /*  9 */ PAUSE_STATE_GAME_OVER_WAIT_BG_PRERENDER,
+    /* 10 */ PAUSE_STATE_GAME_OVER_INIT,
+    /* 11 */ PAUSE_STATE_GAME_OVER_SHOW, // Show the "GAME OVER" message
+    /* 12 */ PAUSE_STATE_GAME_OVER_DELAY, // Wait for the delay
+    /* 13 */ PAUSE_STATE_GAME_OVER_FRAME, // Show background and animate
```

Comment:
Maybe `SHOW_WINDOW`?
When I saw "frame" initially I thought of like an in game frame, like in the frames per second sense.
Im sure the pause menu already has a standard term for this (@Dragorn421 ?)
I also think adding "show" for symmetry with "show message" is nice

Reply:
agreed